Breeze.TumbleBit.Client/TumbleBitManager.cs

```diff
@@ -791,6 +792,35 @@ public void ProcessBlock(int height, Block block)
             this.TumblingState.Save();
         }
 
+        public string TumblingDuration(Money walletBalance, Result<ClassicTumblerParameters> tumblerParameters)
+        {
+
+            /* Tumbling cycles occur up to 117 blocks (endBlock) and overlap every 24 blocks (cycleOverlap) :-
+
+                Start block	End block
+                ----------- ---------
+                0	        117
+                24	        141
+                48	        165
+             */
+            const int endBlock = 117;

```

Comment:
cycleDuration would be a better name for this constant

Reply:
done